packages/mui-material/src/Select/SelectInput.js

```diff
@@ -308,7 +340,13 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
     // if open event.stopImmediatePropagation
     if (!open && onBlur) {
       // Preact support, target is read only property on a native event.
-      Object.defineProperty(event, 'target', { writable: true, value: { value, name } });
+      Object.defineProperty(event, 'target', {
```

Comment:
I can't comment on unchanged code. Few lines above, at line 337, there's no need to have `(isOpenControlled ? openProp : openState)`. `const open = displayNode !== null && openState` will be enough.

Reply:
Didn't catch that. Thanks!